guides/v2.1/javascript-dev-guide/javascript/js_overview.md

```diff
@@ -27,52 +24,18 @@ Topics of this chapter describe the following:
 - [Magento jQuery widgets]({{ page.baseurl }}/javascript-dev-guide/widgets/jquery-widgets-about.html): Magento {% glossarytooltip 5bfa8a8e-6f3e-4fed-a43e-62339916f02e %}jQuery{% endglossarytooltip %} {% glossarytooltip f0dcf847-ce21-4b88-8b45-83e1cbf08100 %}widget{% endglossarytooltip %} {% glossarytooltip 786086f2-622b-4007-97fe-2c19e5283035 %}API{% endglossarytooltip %} documentation.
 - [Customizing JavaScript illustration]({{ page.baseurl }}/javascript-dev-guide/javascript/js_practice.html): practical illustration of custom widgets related tasks.
 
-
-
-## Terms used {#js_terms}
-
-<table>
-<tr>
-<th>
-Term
-</th>
-<th>
-Description
-</th>
-</tr>
-<tr>
-<td>
-<i>JavaScript component (JS component)</i>
-</td>
-<td>
-Any separate <code>.js</code> file decorated as <a href="http://requirejs.org/docs/whyamd.html#amd" target="_blank">AMD module</a>.
-</td>
-</tr>
-
-<tr>
-<td>
-<i>Ui component</i>
-</td>
-<td>
-JS component located in the <code>Magento_Ui</code> module, in the <a href="{{ site.mage2000url }}app/code/Magento/Ui/view" target="_blank">app/code/Magento/Ui/view</a> directory.
-</td>
-</tr>
-
-<tr>
-<td>
-<i>jQuery UI widget</i>
-</td>
-<td>
-A JS component/widget provided by <a href="{{ site.mage2000url }}lib/web/jquery/jquery-ui-1.9.2.js" target="_blank">jQuery UI library used in Magento</a>.
-</td>
-</tr>
-<tr>
-<td>
-<i>jQuery widget</i>
-</td>
-<td>
-Custom widget created using jQuery UI Widget Factory and decorated as AMD {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %}. Many Magento JS components are jQuery widget.
-</td>
-</tr>
-</table>
-
+Terms used {#js_terms}
+
+  Term                                    Description
```

Comment:
Is this supposed to be a table? If so, can you change the syntax to us the more common markdown format.